tools/java/pom.xml

```diff
@@ -118,13 +118,9 @@
 			<plugin>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<configuration>
-					<systemPropertyVariables>
-						<CLDR_DIR>${project.basedir}/../../</CLDR_DIR>
-						<CLDR_ENVIRONMENT>UNITTEST</CLDR_ENVIRONMENT>
-						<java.awt.headless>true</java.awt.headless>
+					<!-- <systemPropertyVariables>
```

Comment:
Again, "Leave No Code in Comments Behind"?

Reply:
I expect to need this, but will remove it for now.

I thought iot better to comment it out,. than to leave it in but unused.